src/EditorFeatures/TestUtilities/StubVsEditorAdaptersFactoryService.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.ComponentModel.Composition;
+using Microsoft.VisualStudio.Editor;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+using Microsoft.VisualStudio.TextManager.Interop;
+using Microsoft.VisualStudio.Utilities;
+using IServiceProvider = Microsoft.VisualStudio.OLE.Interop.IServiceProvider;
+
+namespace Microsoft.CodeAnalysis.Editor.UnitTests
+{
+    [Export(typeof(IVsEditorAdaptersFactoryService))]
+    [PartNotDiscoverable]
+    internal class StubVsEditorAdaptersFactoryService : IVsEditorAdaptersFactoryService
```

Comment:
➡️ Something in the editor assemblies imports this service, and without the service it will fail to include those items in the MEF catalog. I'm not sure which specific change to the tests resulted in the need for this to be included.